tests/functional/disaster-recovery/regional-dr/test_2az_failover_relocate_zone_failure_test.py

```
@@ -0,0 +1,683 @@
+                )
+                logger.info(
+                    f"Failover complete [{wl_meta['idx']}/{len(all_workloads)}]: "
+                    f"drpc_name={wl_meta['drpc_name']}, "
+                    f"cluster={wl_meta['secondary_cluster_name']}, "
+                    f"namespace={wl_meta['workload_namespace']}"
+                )
+        except Exception as ex:
+            logger.exception(
+                f"Failover phase failed at workload "
+                f"{len(completed_failovers)+1}/{len(all_workloads)}: "
+                f"completed={completed_failovers}, batch={failover_batch}"
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)